lib/workers/schedule_worker.rb

```diff
@@ -75,11 +75,10 @@ def worker_setting_or_default(keys, default=nil)
   end
 
   def system_schedule_every(*args, &block)
-    begin
-      @system_scheduler.schedule_every(*args, &block)
-    rescue ArgumentError => err
-      _log.error("#{err.class} for schedule_every with #{args.inspect}.  Called from: #{caller[1]}.")
-    end
+    raise ArgumentError if args.first.nil?
```

Comment:
there was a test for it, but it was leaning on Rufus' raising an `ArgumentError` for passing `nil` to `#schedule_every`, which it no longer does. I added this to preserve that behavior